README.md

```diff
@@ -152,6 +153,10 @@ All modules should contain a `LICENSE` file. Generally, an Apache 2.0 license sh
 
 Once all the above have been done it is expected that modules are deployed to the Puppet Forge under the `ploperations` user. Once that initial push has been done it is also expected that the `Puppetfile` in our control repo pulls from the Forge.
 
+Using the provided GitHub Actions template, a workflow can be manually triggered to tag a release on GitHub and publish the module to The Forge using the version specified in `metadata.json`.
+
+To trigger the workflow navigate to `Actions --> Workflows --> Publish Module --> Run workflow with main branch`.
```

Comment:
Add a bit about what setup is needed in the repo secrets wise to make this work.

Reply:
Added a reference to step 3 in the GitHub UI settings above.